server/planning/tests/__init__.py

```diff
@@ -17,3 +17,15 @@ def setup_test_user(self):
         user = {"_id": ObjectId()}
         self.app.data.insert("users", [user])
         g.user = user
+
+    async def setUp(self):
```

Comment:
If a test cases setup is async, then you use the `async def asyncSetUp(self):` method instead (and make sure to call `await super().asyncSetUp()` as well.

As for the app_context, I thought this was already being done on the [AsyncFlaskTestCase](https://github.com/superdesk/superdesk-core/blob/async/superdesk/tests/__init__.py#L643) from core?

Reply:
you're right. For some reason, it was failing before, because of the missing app context but, I guess we were inheriting from the wrong base class.

Reply:
Done.